src/main/java/frc/robot/Robot.java

```diff
@@ -105,6 +108,28 @@ public void teleopInit() {
    */
   @Override
   public void teleopPeriodic() {
+    String gameData;
+    gameData = DriverStation.getInstance().getGameSpecificMessage();
+    if (gameData.length() > 0) {
+      switch (gameData.charAt(0)) {
+      case 'B':
+        positionControlColor = 'B';
+        break;
+      case 'R':
+        positionControlColor = 'R';
+        break;
+      case 'Y':
+        positionControlColor = 'Y';
+        break;
+      case 'G':
+        positionControlColor = 'G';
+        break;
+      default:
```

Comment:
If we ever reach this default block, there's something wrong with data sent from the FMS so maybe print an error to the console.

Reply:
Based on the experience from 2018, it was common to get a null String.  That should be handled first.  The default clause would be a good place to log any unusual string contents.